### DISCUSSION TITLE
Footnote 5

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#83] [Text comments] [answered]
> NumFOCUS is a 501(c)(3) nonprofit that supports and promotes world-class, innovative, open source
scientific computing.

People outside the US won't know what 501(c)(3) means. I think a less technical wording might be more suitable. Something like "NumFOCUS is a non-profit organization in the United Stats or America, that can collect donations, administer grants, and perform financial management for projects in open source scientific computing."

ApJ is a US-based journal, but the readers are world-wide, so we should try to avoid US-based jargon. (And in writing that, I note the "US based" is already US jargon, as, outside of North America, the country is generally abbreviated as "USA", not "US".)

**Comment by pllim** [comment]
Unless that blurb is the legal wording that NumFOCUS lawyers require? Do we have a NumFOCUS contact to ask about this?

**Reply by hamogu** [reply]
We are writing for a scientific audience. It is true that NumFOCUS is a 501(c)(3) and we may want to say that in documents requesting donations, but here the critical words are "non-profit" and "not a major science institute".

**Reply by adrn** [reply]
I don't see the harm in being specific here. If it was a german company and it said "GmbH", I would just google to find out what that means. I would be fine with:
NumFOCUS is a 501(c)(3) non-profit organization in the United States of America that can collect donations, administer grants, and perform financial management for projects in open source scientific computing.

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
